ext/nnstreamer/tensor_filter/tensor_filter_lua.cc

```diff
@@ -70,17 +74,97 @@ void _fini_filter_lua (void) __attribute__((destructor));
 }
 #endif /* __cplusplus */
 
+/** @brief For getting value in Lua */
+static int
+tensor_index (lua_State *L)
+{
+  uint8_t** parray = (uint8_t **) luaL_checkudata (L, 1, "tensor");
+  int index = luaL_checkint (L, 2);
+  lua_pushnumber (L, (*parray)[index - 1]);
+
+  return 1;
+}
+
+/** @brief For assigning new value in Lua */
+static int
+tensor_newindex (lua_State* L)
+{
+  uint8_t** parray = (uint8_t **) luaL_checkudata(L, 1, "tensor");
+  int index = luaL_checkint(L, 2);
+  int value = luaL_checkint(L, 3);
+  (*parray)[index - 1] = value;
```

Comment:
is it okay to not check `index - 1` goes negative?

Reply:
Thanks! I'm preparing the index checking in the following PR.